musicbrainz_mmd.xsd

```diff
@@ -1275,6 +1341,7 @@
           <xs:sequence>
             <xs:element maxOccurs="unbounded" name="attribute">
               <xs:complexType mixed="true">
+                <xs:attribute name="type-id" use="required" type="mmd-2.0:def_uuid"/>
```

Comment:
Just in case someone else is wondering: This is the id of relationships that was first added in https://github.com/metabrainz/mmd-schema/pull/16 and renamed in https://github.com/metabrainz/mmd-schema/commit/8600db45a4c61c3e8d8b0c0cdc6ceff7a7a94919.

Reply:
For the record, it refers to [MBS-9470](https://tickets.metabrainz.org/browse/MBS-9470).